### PR TITLE
AOM-129: Release Open Web Application

### PR DESCRIPTION
**Pull request by patlub**
## JIRA TICKET NAME:
[AOM-129: https://issues.openmrs.org/browse/AOM-129](https://issues.openmrs.org/browse/AOM-129)

### SUMMARY:

Working on the steps required to Release an Open Web Application

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,15 @@
...
+deploy:
+  provider: bintray
+  file: bintray.json
+  user: patlub
+  dry-run: false
+  on:
+    tags: true
+    branch: master
+  key:
+    secure: GWzNVyqkmJpme05DDuBCk51OUjFpSg1Yv1CC40vXFlVR48pYD0kiHZ3vtrvTMGeXONFanClzRQpqsbQ0y/q3QlUHP6UBwCu7K8zFt4QOMkOlRdjoeNThHFWKKSRtkbMaIRUwmOWwh3IIQyVkYr3LtvfjL0qBzXjFBBl5MTmU+n2k2HHed+j4G1TiXbT1ZCxc7vYSkxhKLvBbTDBCuuxvFD5LQ4VX65IEL98SDNaJk9MyhHubQCacUVwLPE6NjrXg9Y73dkx5IpCoyUON/8uLdiLJtXv6EXjKyA449zxp6NCN8jm0p93Wd3MgD0HUdl7yIlvbwSaVe0YhfgYBq8gsKbEEawHqT0N1sNLNBP+tHbk3r31N0+CBdhOkVUU1XxQjob5KfRsWTo9WaIei6Md3N5NaUh7EOD1pqv6AI3yHwueRHL9TvLUBqm1AXp8vIkxT5gT806/GwUfVipgai7JpM6WANA5z9FLNx4bn8VjwEZwrSU9bf554SCY0eamH1AOAsOd6bBgMEZ8naHFZ0zqhCOwDca9SFOkCje3DWe5DlXeCrYZidn5sN4GgjtWDY7lqqasPwdPdk66AsVjFXwnSIQMP54nPuOhfRqn3sP5el9xwDO3nZx384v+6NEFVdaBAjmfFp0AI9+XN/eU4dncLOEZ55hDwvAvnMk1r8lX6ov0=
```
Comment (segunolalive) [inline .travis.yml:15]: could this key be stored in an environment variable?

**Review by segunolalive — COMMENTED** [review]
Looks good

File: .travis.yml (hunk 1 above)
Comment (patlub) [reply .travis.yml:15]: It's encrypted(it's not the real key, that's a secure hash), so no need for that.

**Comment by Annettesunday** [comment]
Good job 👍

**Review by malmike — COMMENTED** [review]
Looks good to me

**Pull request merged**